include/client/libcurve.h

```diff
@@ -474,7 +480,7 @@ class CurveClient {
      * @return 成功返回fd，失败返回-1
      */
     virtual int Open(const std::string& filename,
-                     std::string* sessionId);
+                     const OpenFlags& openflags);
```

Comment:
Why is sessionId not needed here？

Reply:
session has been removed, so it became useless.